pubspec.yaml

```diff
@@ -13,7 +13,7 @@ description: A mobile application for students at ACS UPB.
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 #
 # ACS UPB Mobile uses semantic versioning. You can read more in the CONTRIBUTING.md file.
-version: 1.2.13+27
+version: 1.3.0+28
```

Comment:
Let's make it 1.2.14, and I'll create a release with 1.3.0 once we merge the other PRs that are done.